### PR TITLE
ListToStringConverter

### LINKED ISSUE #238: [Enhancement] ListToStringConverter
## Summary
Convert enumerable of objects (strings) to the single string. Combine "ToStrings" representations into a single string with separator.

```xaml
<Label Text="{Binding Errors, Converter={StaticResource ListToStringConverter}, ConverterParameter={StaticResource CommaWithSpaceSeparator}}" />
```
Input
```["error1", "error2", "error3"]```

Output
```error1, error2, error3 ```

### PR DESCRIPTION
**Pull request by AndreiMisiukevich**
### Description of Change ###

Added ListToStringConverter

```xaml
<Label
    Text="{Binding Collection, Converter={converters:ListToStringConverter Separator=','}}" />
```

### Bugs Fixed ###

- Fixes #238 

### API Changes ###

List all API changes here (or just put None), example:

Added: 
- `class ListToStringConverter`

### Behavioral Changes ###
None

### PR Checklist ###

- [X] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [X] Rebased on top of main at time of PR
- [X] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

**Review by sthewissen — APPROVED** [review]

File: XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Concatenates the members of a collection, using the specified separator between each member.
+	/// </summary>
+	public class ListToStringConverter : ValueConverterExtension, IValueConverter
+	{
+		/// <summary>
+		/// The separator that should be between each coolection member.
```
Comment (jfversluis) [inline XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs:16]:
```suggestion
		/// The separator that should be between each item in the collection
```

File: XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		/// </summary>
+		public string Separator { get; set; }
+
+		/// <summary>
+		/// Concatenates the members of a collection, using the specified separator between each member.
```
Comment (jfversluis) [inline XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs:21]:
```suggestion
		/// Concatenates the items of a collection, using the specified separator between each item.
```

File: XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		/// </summary>
+		/// <param name="value">The collection to convert.</param>
+		/// <param name="targetType">The type of the binding target property.</param>
+		/// <param name="parameter">The separator that should be between each coolection member.</param>
```
Comment (jfversluis) [inline XamarinCommunityToolkit/Converters/ListToStringConverter.shared.cs:25]:
```suggestion
		/// <param name="parameter">The separator that should be between each collection item.</param>
```

**Review by jfversluis — APPROVED** [review]
Some spelling things and while fixing those I found "item" a bit better fitting than "member" but maybe that's just me :)

**Comment by AndreiMisiukevich** [comment]
@jfversluis updated)

**Review by sthewissen — APPROVED** [review]

**Pull request merged**